lib/matplotlib/contour.py

```diff
@@ -1259,12 +1259,16 @@ def draw(self, renderer):
             super().draw(renderer)
             return
         # In presence of hatching, draw contours one at a time.
+        all_edgecolors = self.get_edgecolors()
```

Comment:
```suggestion
        edgecolors = self.get_edgecolors()
```
Slight preference for leaving out the `all_` prefix. It does not convey information.